__std__/flake.lock

```diff
@@ -3,11 +3,28 @@
     "CHaP": {
       "flake": false,
       "locked": {
-        "lastModified": 1670900592,
```

Comment:
adding the cardano-node flake added ~9000 lines to flake.lock. Not sure how to avoid this?